OAuthSwift/OAuthSwiftHTTPRequest.swift

```diff
@@ -54,7 +60,7 @@ public class OAuthSwiftHTTPRequest: NSObject, NSURLSessionDelegate {
         self.init(URL: URL, method: .GET, parameters: [:])
     }
 
-    init(URL: NSURL, method: Method, parameters: Dictionary<String, AnyObject>) {
+    init(URL: NSURL, method: Method, parameters: Dictionary<String, AnyObject>, paramsLocation : ParamsLocation? = .AuthorizationHeader) {
```

Comment:
`ParamsLocation?` could become `ParamsLocation` and so no need to do unwrap : `paramsLocation!`
unwrap is dangerous

in `setupRequestForOAuth` and `makeRequest` too